assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -111,11 +52,91 @@ const cartTotals = {
  * Component that renders the Cart block when user has something in cart aka "full".
  */
 const Cart = () => {
+	const [ selectedShippingRate, setSelectedShippingRate ] = useState();
+	const [
+		shippingCalculatorAddress,
+		setShippingCalculatorAddress,
+	] = useState( {
+		city: '',
+		county: '',
+		postcode: '',
+		country: '',
+	} );
+
+	/**
+	 * Given an API response with cart totals, generates an array of rows to display in the Cart block.
+	 *
+	 * @param {Object} cartTotals - Cart totals data as provided by the API.
+	 * @return {Object[]} Values to display in the cart block.
+	 */
+	const getTotalRowsConfig = () => {
+		const totalItems = parseInt( cartTotals.total_items, 10 );
```

Comment:
Just a question; do we have to be aware of MAX_SAFE_INTEGER if parsing totals like this, or non issue?

Reply:
Interesting point. Did you have anything in mind to do here? I thought we could create a util function that parses the integer and verifies it's between the limits. Something like:
```JS
const safelyParseInt = ( val ) => Math.max( Math.min( parseInt( val, 10 ), Number.MAX_SAFE_INTEGER ), Number.MIN_SAFE_INTEGER );
```
But wondering if we should instead return `null` or throw an error if the parsed value is outside the limits? :thinking:

Reply:
I'm not sure what would be best here. I just mentioned it because the API returns strings to prevent large ints breaking responses. Follow up?

Reply:
Let's create an issue as a followup for this. It's a more general thing that probably will be needed anywhere the values are used in a way that `MAX_SAFE_INTEGER` comes into play.  However, I think it's edgecase enough that it should be non-blocking for the initial iteration of the blocks and something we can do once the main work is done?

Reply:
Issue created: #1754.